src/runtime_filesystem.cpp

```
@@ -15,6 +15,9 @@
 
 #ifdef __APPLE__
 #include <copyfile.h>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)